src/app/components/organisms/cart_items/index.tsx

```diff
@@ -5,18 +5,17 @@ import { RecAndDot } from "@app/components/atoms/rectangleAndDot";
 import { ImageContainer, PageWrapper } from '@app/components/ui/PageElement';
 import { useCart } from '@app/hooks/useCarts';
 import Image from "next/image";
-import Link from 'next/link';
 import React from "react";
-import { MdArrowBack } from 'react-icons/md';
 import styled from "styled-components"
 import CartContent from '../cartContent';
 import { LikeButton } from '@app/components/atoms/like';
 import { format_price } from '@app/utils/helper';
 import { Button } from '@app/components/atoms/Buttons';
 import { TProductTesting } from '@app/constant/details';
-import toast from 'react-hot-toast';
 import ProductCategoryCard from '@app/components/atoms/ProductCategoryCard';
 import AddToCartButton from '@app/components/atoms/AddToCartButton';
+import useNotify from '@app/hooks/useNotify';
+import EmptyCart from '../empty_cart';
```

Comment:
```suggestion
import EmptyCart from '@app/components/organisms/empty_cart';
```

And please use PascalCase for naming folders . So the folder name should be EmptyCart. Make this changes in other areas needed.